src/components/Chat/Chat.tsx

```diff
@@ -56,6 +56,10 @@ export const Chat: React.FC<Props> = ({ talk }) => {
     return api
       .apiV1ChatMessagesGet('cndo2021', String(talk?.id), 'talk')
       .then((res) => {
+        if (typeof res.data !== 'object') {
+          // Chatのwebsocketがエラーを返した場合はログインさせるためにトップページへリダイレクト
+          window.location.href = `${window.location.href.replace(/ui/g, '')}`
```

Comment:
Review appsで偶然誤爆が起きて気づいたのだけど、
```
https://dreamkast-ui-54.dev.cloudnativedays.jp/cndo2021/ui/
```
だと
```
https://dreamkast--54.dev.cloudnativedays.jp/cndo2021//
```

にリダイレクトされるという状態になりました。

```suggestion
          window.location.href = `${window.location.href.replace(/\/ui\//g, '')}`
```

とかどうでしょ

Reply:
そのほうがよさそうですね！取り入れます